modules/openshift-developer-cli-sample-applications.adoc

```diff
@@ -24,7 +24,13 @@ ruby        openshift     2.0,2.2,2.3,2.4,latest
 wildfly     openshift     10.0,10.1,8.1,9.0,latest
 ----
 
-In order to access the component over the web, you can create a URL using `odo url create`.
+[NOTE]
+====
+For {odo-title}{nbsp}{odo-ver} Java and Node.js are the officialy supported component types.
```

Comment:
This is the first and only time that the version (1.0) is used. Just want to make sure that at some point this will be referenced somewhere else (like the release notes you were mentioning?)

Reply:
The release notes and KIs for sure.
The version attribute will always come in handy at some point. :D

Reply:
officialy -> officially

Reply:
Typo fixed.